packages/create-svelte/templates/default/src/routes/todos/index.svelte

```diff
@@ -78,12 +94,13 @@
 			<form class="text" action="/todos/{todo.uid}.json?_method=patch" method="post" use:enhance={{
 				result: patch
 			}}>
-				<input type="text" name="text" value={todo.text}>
+				<input aria-label="Edit todo" type="text" name="text" value={todo.text} bind:this={todoInputs[i]}>
 				<button class="save" aria-label="Save todo"/>
 			</form>
 
 			<form action="/todos/{todo.uid}.json?_method=delete" method="post" use:enhance={{
-				result: () => {
+				result: async () => {
+					await manageFocus(i);
```

Comment:
Don't we want to manage focus _after_ the todo has been removed from the DOM?

Reply:
Removing explicit focus management entirely per Rich's comments.